src/desktop/apps/artist/components/cta.ts

```diff
@@ -4,22 +4,27 @@ import { handleScrollingAuthModal } from "desktop/lib/openAuthModal"
 import { Intent, ContextModule } from "@artsy/cohesion"
 const Cookies = require("desktop/components/cookies/index.coffee")
 const mediator = require("desktop/lib/mediator.coffee")
-const metaphysics = require("lib/metaphysics.coffee")
+const metaphysics2 = require("lib/metaphysics2.coffee")
 
 export const query = `
 query ArtistCTAQuery($artistID: String!) {
   artist(id: $artistID) {
     name
-    artworks(size: 1) {
-      image {
-        cropped(width: 390, height: 644) {
-          url
+    filterArtworksConnection(first: 1, marketable: true) {
```

Comment:
We should also add an explicit `sort` value here!

It looks like the previous endpoint, by default, sorts based on the artwork's [iconicity](https://github.com/artsy/gravity/blob/master/app/api/v1/artists_artworks_endpoint.rb#L33), but by default the filter endpoint (which we're now calling), will sort by [`partner_updated_at`](https://github.com/artsy/gravity/blob/master/app/api/v1/filter_endpoint.rb#L41).

In this case, I think it's 👍 if we stick a `sort: "-decayed_merch"` in the args here.

That'll give us a more iconic artwork like
![image](https://user-images.githubusercontent.com/2081340/81019286-5abcfd00-8e34-11ea-849e-96c88d2f227f.png)
😄